tests/test.py

```diff
@@ -89,74 +89,74 @@ def test_post_graph(name=None):
 	})
 	graph1.set_tags(['sample'])
 	response = graphspace.post_graph(graph1)
-	assert 'name' in response and response['name'] == graph1.get_name()
+	assert hasattr(response, 'graph') and response.graph.name == graph1.get_name()
 
 
 def test_get_graph(name):
 	graphspace = GraphSpace('user1@example.com', 'user1')
 	graphspace.set_api_host('localhost:8000')
-	graph = graphspace.get_graph(name)
-	assert graph is not None and graph['name'] == name
+	response = graphspace.get_graph(name)
+	assert hasattr(response, 'graph') and response.graph.name == name
 	return graph
 
 
 def test_get_graph_by_id():
 	graphspace = GraphSpace('flud', 'Muraliistheman!')
 	# graphspace.set_api_host('localhost:8000')
-	graph = graphspace.get_graph_by_id(20047)
-	assert graph is not None and 'id' in graph
+	response = graphspace.get_graph_by_id(20047)
+	assert hasattr(response, 'graph') and response.graph.id == 20047
 
 
 def test_get_public_graphs():
 	graphspace = GraphSpace('user1@example.com', 'user1')
 	graphspace.set_api_host('localhost:8000')
 	response = graphspace.get_public_graphs(tags=['Kegg-networks'])
-	assert response is not None and len(response['graphs']) > 0
+	assert hasattr(response, 'graphs') and len(response.graphs) > 0
 
 
 def test_get_shared_graphs():
 	graphspace = GraphSpace('user1@example.com', 'user1')
 	graphspace.set_api_host('localhost:8000')
 	# response = graphspace.get_public_graphs(tags=['2015-bioinformatics-xtalk', 'kegg-curated-top-rank-FPs'])
 	response = graphspace.get_shared_graphs()
-	assert response is not None and len(response['graphs']) > 0
+	assert hasattr(response, 'graphs') and len(response.graphs) > 0
 
 
 def test_get_my_graphs():
 	graphspace = GraphSpace('user1@example.com', 'user1')
 	graphspace.set_api_host('localhost:8000')
 	# response = graphspace.get_public_graphs(tags=['2015-bioinformatics-xtalk', 'kegg-curated-top-rank-FPs'])
 	response = graphspace.get_my_graphs()
-	assert response is not None and len(response['graphs']) > 0
+	assert hasattr(response, 'graphs') and len(response.graphs) > 0
 
 
 def test_get_my_graph_layouts(graph_id):
 	graphspace = GraphSpace('user1@example.com', 'user1')
 	graphspace.set_api_host('localhost:8000')
 	response = graphspace.get_my_graph_layouts(graph_id=graph_id)
-	assert response is not None and len(response['layouts']) >= 0
+	assert hasattr(response, 'layouts') and len(response.layouts) >= 0
```

Comment:
response should be of type Layout class object (add a check for that too in the test code).